packages/tegh-host-posix/scripts/tegh-server.service

```diff
@@ -8,7 +8,5 @@ ExecStart=/usr/sbin/tegh-host-posix
 Type=simple
 Restart=on-failure
 
-User=
```

Comment:
The User directive no longer applies as the service unit is now run by the systemd user instance.